docs/superpowers/plans/2026-08-02-registry-ai-discoverability.md

```
@@ -0,0 +1,2490 @@
+git status --porcelain packages/cli/src/utils/fallback-generated.ts
+cp packages/cli/src/utils/fallback-generated.ts /tmp/fallback-reference.ts
+```
+
+Expected: `git status` prints nothing (the committed file already matches the generator).
+If it prints a modification, commit that first — you need a clean baseline.
+
+- [ ] **Step 2: Un-ignore the `registry/` directory**
+
+Open `.gitignore` and delete these five lines from the end of the file:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)